### PR TITLE
add focus to bioconda

### PR DESCRIPTION
**Pull request by metageni**
* [ x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [ x] This PR adds a new recipe.
* [ x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/focus/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  home: https://edwards.sdsu.edu/FOCUS
+  license: GPL-3.0
+  license_family: GPL
+  license_file: COPYING
+  summary: 'FOCUS is an innovative and agile model to profile and report organisms present in metagenomic samples based on composition usage without sequence length dependencies.'
+  dev_url: https://github.com/metageni/FOCUS
+
+extra:
+  skip-lints:
+    - should_be_noarch
```
Comment (bgruening) [inline recipes/focus/meta.yaml]: why did you skipped it and not used noarch?

Comment (metageni) [reply recipes/focus/meta.yaml]: @bgruening  Fixed it. Let me know if it good to go now. Thanks

Comment (metageni) [reply recipes/focus/meta.yaml]: @bgruening Fixed it. Let me know if it good to go now. Thanks

**Comment by bgruening** [comment]
Thanks!

**Pull request merged**